github/apps_marketplace.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2017 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// MarketplaceService handles communication with the marketplace related
+// methods of the GitHub API.
+//
+// GitHub API docs: https://developer.github.com/v3/apps/marketplace/
+type MarketplaceService struct {
+	client *Client
+	// Stubbed controls whether endpoints that return stubbed data are used
+	// instead of production endpoints. Stubbed data is fake data that's useful
+	// for testing your GitHub Apps. Stubbed data is hard-coded and will not
+	// change based on actual subscriptions.
+	Stubbed bool
```

Comment:
This `Stubbed` field is quite unusual and may cause people to seek more information about it. I think it's worth it to include a link to GitHub API documentation for it as well here, as I suggested in https://github.com/google/go-github/pull/732#issuecomment-333726743:

```diff
 // Stubbed controls whether endpoints that return stubbed data are used
 // instead of production endpoints. Stubbed data is fake data that's useful
 // for testing your GitHub Apps. Stubbed data is hard-coded and will not
 // change based on actual subscriptions.
+//
+// GitHub API docs: https://developer.github.com/v3/apps/marketplace/
 Stubbed bool
```

Reply:
@gmlewis Does that sound good to you?

If so, I can apply it and merge.